### PR TITLE
feature: add envd top commands

### PR DESCRIPTION
**Pull request by aseaday**

### CODE REVIEW

File: pkg/docker/stats_type.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
+// The following code is from https://github.com/fsouza/go-dockerclient/
+// Copyright (c) 2013-2021, go-dockerclient authors
+package docker
+
+import "time"
+
+// Stats represents container statistics, returned by /containers/<id>/stats.
+// See https://goo.gl/Dk3Xio for more details.
+type Stats struct {
```
Comment (gaocegege) [inline pkg/docker/stats_type.go:9]: Do we need to maintain the struct in envd?

**Comment by aseaday** [comment]
for future gpu support which go docker doesn't support now.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: pkg/docker/stats_type.go (hunk 1 above)
Comment (aseaday) [reply pkg/docker/stats_type.go:9]: It is also for future support for GPU. Another point I cared about  is that the godocker repo's author has no GPG keys verified sign which could cause potienal danger.

**Comment by aseaday** [comment]
image

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Pull request merged**